rticommunity/rticonnextdds-connector-py
# COMMUNITY-100 | Create new CI pipelines for Jenkins

This PR is to add new pipelines to validate connector-py using Jenkins.
The build_and_test pipeline will do the following:
1. Download connext libs.
2. Run tests on Python 2.7 and 3.6.
3. Publish package to our internal pypi server if a tag with format X.Y.Z-dev is pushed (this package can be overwritten without any problem).

The build_doc pipeline will do the following:
1. Download connext libs (it sounds weird but it seems to be necessary).
2. Install python dependencies.
3. Build doc and store the artifacts on the pipeline.
4. Publish doc if a tag with format X.Y.Z-doc is pushed.

Keep in mind that this connector is in maintenance not in development, so we should not work too much on the pipeline.
